pkg/apis/label/label.go

```diff
@@ -206,6 +214,10 @@ const (
 	ApplicationLabelKey string = "app.kubernetes.io/app"
 )
 
+func PDMSLabel(name string) string {
+	return name
+}
```

Comment:
I was a little confused until I looked at where it was called. Would you be willing to add some comments?